src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -3101,3 +3112,41 @@ MavlinkReceiver::receive_start(pthread_t *thread, Mavlink *parent)
 
 	pthread_attr_destroy(&receiveloop_attr);
 }
+
+float
+MavlinkReceiver::get_offb_cruising_speed()
+{
+	vehicle_status_s vehicle_status{};
+	_vehicle_status_sub.copy(&vehicle_status);
+
+	if (vehicle_status.vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING) {
```

Comment:
@ThomasRigi this could be compressed a bit. How about 
```suggestion
	if (vehicle_status.vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING && _offb_cruising_speed_mc > 0.0f) {
	return _offb_cruising_speed_mc;
	} else if (vehicle_status.vehicle_type == vehicle_status_s::VEHICLE_TYPE_FIXED_WING && _offb_cruising_speed_fw > 0.0f) {
	return _offb_cruising_speed_fw;
	} else {
	return -1.0f;
	}
```

Reply:
Thanks for the suggestion!